src/v2/controllers/FormController.js

```diff
@@ -167,6 +171,10 @@ export default Controller.extend({
   },
 
   showFormErrors (model, error) {
+    // If errors are terminal errors, then clear session storage, so that intentional hard refersh initiate a new loginflow
+    if(error.messages) {
```

Comment:
While testing in November, I figured there are some flows that ends up with form errors in a way that only thing you can do is to hard refresh the page.. I don't exactly remember what was the flow, but happens when user is locked out but still trying to login or password needs to be reset. If i came across that again i will update. But this is required.

Reply:
Are you sure that whenever `error.messages` exists, it's terminal error?
I had vague memory that `showFormErrors` is general error handler (e.g. in case of wrong password, it invokes `showFormErrors` to render form level error message.)

Reply:
Yeah, if error is at top level it is should be terminal state error. All form level errors will be present inside remediation object at field level.

Reply:
Here one example I was worrying. Please correct me wrong.

1. introspect gets `authenticator-verification-email.json`
2. user enter wrong OTP code 
3. server 403 with `error-email-verify.json`
4. widget invokes this `showFormErrors` to show form error and the error has `error.messages`

Don't think we shall clear session storage for this case, right?

Reply:
It doesn't.. But i see your point, we need a better way to cleanup storage from a single source.. i am exploring postRender suggestion.. 

![Screen Shot 2021-01-12 at 10 31 36 AM](https://user-images.githubusercontent.com/38230587/104357182-b5c5ea00-54c1-11eb-9358-330cbca4a2bc.png)

Reply:
removed it.